### ISSUE TITLE
Lets be a bit more careful/cleaner with the git history and best practices

### ISSUE THREAD

**Issue by elbenfreund** [Dezentrale-eV/dezentrale_web#52]
In order to enforce a clean and concise git history I would offer to merge all requested PRs myself **if that** is wanted. As this is no viable long term solution I would also suggest a private personal meeting to have a first gentle introduction into git best practices and our branching model.

Please note: I will take no offence if you want to keep going like this, but I find it increasingly hard to follow the code/issue/PR paths and I feel it is important to establish a rigorous workflow right from the start.

**Comment by bocian67** [comment]
+1

**Comment by Rascat** [comment]
:+1: would it be possible for you guys to meet this week?

**Comment by elbenfreund** [comment]
Yes. Thu and onwards would be best,but I can accomodate.

**Comment by Rascat** [comment]
Thursday in the evening would suit me. 18:00h?

**Comment by bocian67** [comment]
I'll be there too! I invited Lokke to join us and he said he will be
joining.

Am 23.10.2017 19:32 schrieb "Rascat" <notifications@github.com>:

**Comment by elbenfreund** [comment]
I am sorry guys, I will not be able to be there before 20:00 :(
This just came up, so I am really sorry for the lack of notice...
Please let me know if you still want to meet.

**Comment by Rascat** [comment]
20:00 is fine for me as well. Guess i'll be there at 19h - open end

**Comment by bocian67** [comment]
I'll be there at 7 as well, but let's talk about it quite fast since
tomorrow I have to take a test and cannot be there longer than 9:30.

Am 26.10.2017 11:18 vorm. schrieb "Rascat" <notifications@github.com>:

**Comment by Rascat** [comment]
thanks again for the input @elbenfreund ! I am going to close this issue now since our meeting has already taken place.
